### PR TITLE
@ #20 | update README file

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #20

### CODE REVIEW

**Comment by hieptranquoc** [comment]
@hoatle fixed all!

**Comment by hoatle** [comment]
@duc4nh should help with English grammar checking

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Kubernetes stack cookbook to work with Kubernetes
 | centos-7     | ✔      | ✔       | ✔    |
 | ubuntu-16.04 | ✔      | ✔       | ✔    |
 
-- [kubectl](#kubectl): support all version with centos-7 and ubuntu-16.04.
-- [helm](#helm): support all version with centos-7 and ubuntu-16.04.
-- [gcloud](#gcloud): support all version with centos-7 and ubuntu-16.04. Should use version avaiable in https://packages.cloud.google.com/apt/ (with ubuntu platform) for faster autocomplete.
+- `kubectl`: support all version with centos-7 and ubuntu-16.04.
```
Comment (ducng0610) [inline README.md]: support all centos-7 and ubuntu-16.04 versions.

File: README.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ Kubernetes stack cookbook to work with Kubernetes
...
+- `helm`: support all version with centos-7 and ubuntu-16.04.
```
Comment (ducng0610) [inline README.md]: support all centos-7 and ubuntu-16.04 versions. <- with `s`

File: README.md [hunk 3]
```diff
@@ -13,9 +13,9 @@ Kubernetes stack cookbook to work with Kubernetes
...
+- `gcloud`: support all version with centos-7 and ubuntu-16.04. Should use version avaiable in https://packages.cloud.google.com/apt/ (with ubuntu platform) for faster autocomplete.
```
Comment (ducng0610) [inline README.md]: support all centos-7 and ubuntu-16.04 versions.

File: README.md [hunk 4]
```diff
@@ -25,20 +25,20 @@ Kubernetes stack cookbook to work with Kubernetes
...
 end
 
 helm 'install helm' do
   action [:install, :remove]
-  version ''
-  binary_path '' #application path (if empty, default:/usr/local/bin/helm)
+  version '' #application version (if empty, default: latest)
+  binary_path '' #application path (if empty, default: /usr/local/bin/helm)
 end
 ```
```
Comment (duc4nh) [inline README.md]: Fork this project into `the` your personal account. <- remove `the`

**Review by ducng0610 — DISMISSED** [review]
review done, just need some little changes

File: README.md (hunk 4 above)
Comment (ducng0610) [inline README.md:44]: `Fork this project into the your personal account.` <- remove `the` (line 49)

**Comment by ducng0610** [comment]
Review done

File: README.md (hunk 1 above)
Comment (hieptranquoc) [reply README.md]: fixed!

File: README.md (hunk 2 above)
Comment (hieptranquoc) [reply README.md]: fixed!

File: README.md (hunk 3 above)
Comment (hieptranquoc) [reply README.md]: fixed!

File: README.md (hunk 4 above)
Comment (hieptranquoc) [reply README.md:44]: fixed!

**Comment by hieptranquoc** [comment]
@hoatle updated!

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**